opm/material/eos/PengRobinson.hpp

```diff
@@ -205,27 +205,25 @@ class PengRobinson
             Evaluation VmCubic = max(1e-7, Z[0]*RT/p);
             Vm = VmCubic;
 
-            // find the extrema (if they are present)
```

Comment:
It might take some time before we can discard all these code. As a temporary approach. I added a boolean template argument to make both the old code and new PTFlash work.